### ISSUE TITLE
Bug 1279909 - Cannot browse through Activity Stream context menu using the keyboard

### ISSUE THREAD

**Issue by Softvision-VladBacia** [mozilla/activity-stream#842]
via https://bugzilla.mozilla.org/show_bug.cgi?id=1279909

<hr>

[Affected versions]:
- Firefox 45.0 and up
- Activity Stream 1.1.0-dev

[Affected Platforms]:
- All Windows
- All Linux
- All Mac OS

[Prerequisites]:
- Latest Activity Stream add-on version (1.1.0) was previously installed on a profile.
- Have enough history so that the "Top Sites", "Highlights", "History" areas are populated with visited websites.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Click the Activity Stream button from the browser toolbar.
3. Hover an item from Timeline History and expand the "..." context menu.
4. Press "up arrow" and "down arrow" buttons from keyboard.

[Expected result]:
- The context menu can be navigated using the keyboard buttons

[Actual result]:
- The page is scrolled up and down. The options from the context menu cannot be accessed using the keyboard.

[Notes]:
- When pressing the "Esc" key from keyboard, the context menu should be closed if it is displayed.
- "..." button it is not displayed for items from Activity Stream, if they are selected using the "Tab" keyboard button.
- [Here](https://goo.gl/zr17eC) is a recording with this behaviour.

**Comment by sarracini** [comment]
This is fixed in the system-addon, and we won't be fixing this in the test pilot version
